source/core/src/main/com/csse3200/game/ui/DialogueBox/DialogueBox.java

```diff
@@ -101,29 +106,62 @@ public DialogueBox(String[][] labelText) {
 
     /**
      * Creates the background image.
-     *
-     * @return The Image instance.
      */
-    private Image createBackgroundImage() {
-        float desiredHeight = 700;
-        float imageWidth = BACKGROUND_TEXTURE.getWidth();
-        float imageHeight = BACKGROUND_TEXTURE.getHeight();
-        float ratio = imageWidth / imageHeight;
-        float newWidth = desiredHeight * ratio;
-
-        Image backgroundImage = new Image(new TextureRegionDrawable(BACKGROUND_TEXTURE));
-        backgroundImage.setSize(newWidth, desiredHeight);
-        backgroundImage.setPosition((screenWidth - newWidth) / 2, -125);
-
-        return backgroundImage;
+    private void createBackgroundImage() {
+        backgroundImage = new Image(new TextureRegionDrawable(BACKGROUND_TEXTURE));
+
+        float desiredHeight = screenHeight * 0.08f;  // 7% of the screen height
+        float newWidth = screenWidth;  // Full screen width
+
+        backgroundImage.setSize(newWidth * 0.9f, desiredHeight);
+        backgroundImage.setPosition(newWidth * 0.05f, screenHeight * 0.09f);
+    }
+
+    /**
+     * Resizes the background image for formatting
+     */
+    public void resizeElements() {
```

Comment:
should also resize text in the dialogue box

Reply:
It does on line 139 - Did you have issues with this?

Reply:
Yeah, the actual font in the dialogue box (or buttons) dont resize

Reply:
Yep, heard loud and clear. I'll work on this now.